EnglishFilter/sections/specific.txt

```diff
@@ -292,6 +292,7 @@ techpowerup.com##.site-headr__logo-wrapper + * a[href*=".png"] > img
 techpowerup.com##div[class*="bar-item"] a[href*=".png"]
 techpowerup.com#%#//scriptlet('remove-attr', 'wdrwgbethte', 'div')
 !
+downace.com##.tm-content > .uk-panel:not(.uk-panel-box)
```

Comment:
This rule also removes a useful element.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/144134603-463613cb-db61-4f5e-932d-a5addc4213cf.png)

</details><br/>

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/101403/commits/b600b7f5c3a9d9b055efa1ebc0d9cf299bc3590a